poolrpc/trader.proto

```diff
@@ -370,6 +380,29 @@ message DepositAccountResponse {
     bytes deposit_txid = 2;
 }
 
+message RenewAccountRequest {
+    // The key associated with the account to renew.
+    bytes account_key = 1;
+
+    oneof account_expiry {
+        // The new absolute expiration height of the account.
+        uint32 absolute_expiry = 2;
+
+        // The new relative expiration height of the account.
+        uint32 relative_expiry = 3;
```

Comment:
I think it's enough to only support absolute heght.

Reply:
Hm, or maybe it's useful for the client to not have to learn the current block height?

Reply:
Correct. This is already the case when opening new accounts as well. Communication between the trader and auctioneer use absolute block heights.